formatting/action.yml

```diff
@@ -2,78 +2,164 @@ name: 'formatting'
 description: 'CI formatting check'
 inputs:
   path:
-    description: 'Path to repository folder to run formatting check for.'
+    description: 'Path to repository folder to run formatting check for. '
     required: false
     default: ./
   exclude-files:
-    description: 'List of comma-separated files to exclude from trailing whitespace check. Eg file1,file2'
```

Comment:
Whitespace check was also affecting non-C files, so this no longer checks that those do not have trailing whitespace

Reply:
Good catch, added this in